pkg/azure/credential/ucpcredentials.go

```diff
@@ -171,14 +171,21 @@ func refreshAzureWorkloadIdentityCredentials(ctx context.Context, c *UCPCredenti
 
 	logger.Info("Retrieved Azure Credential - ClientID: " + azureWorkloadIdentityCredential.ClientID)
 
-	var opt *azidentity.DefaultAzureCredentialOptions
+	var opt *azidentity.WorkloadIdentityCredentialOptions
```

Comment:
I misunderstood what this code did on the first workload identity PR. this should have been WorkloadIdentityCredential instead.